File: cpp/0034.cpp
class Solution
{
public:
    vector<int> searchRange(vector<int> &nums, int target)
    {
        int n = nums.size();
        for (int i = 0; i < n; i++)
        {
            if (nums[i] == target)
            {
                int first = i;
                int second = first;
                while (second + 1 < n && nums[second + 1] == target)
                    second++;
                return {first, second};
            }
        }
        return {-1, -1};
    }
};

File: py/0034.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        try:
            length = len(nums)
            first = nums.index(target)
            second = first
            while second+1 < length and nums[second+1] == target:
                second = second + 1
            return [first, second]
        except:
            return [-1, -1]
